src/cms/api/api.ts

```diff
@@ -103,11 +102,16 @@ apiConfig.forEach((entry) => {
     const d1 = getD1Binding(ctx);
 
     try {
-      // console.log("posting new record content", JSON.stringify(content, null, 2));
-
-      const result = await insertRecord(d1, ctx.env.KVDATA, content);
-
-      return ctx.json(result.data, 201);
+      console.log(
+        "posting new record content",
+        JSON.stringify(content, null, 2)
+      );
+      if (table === "users") {
```

Comment:
All of the auth routes can have their own custom hardcoded routes since they won't need to be dynamic